src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1565,6 +1565,16 @@ protected async Task<DebugStore> RuntimeReady(SessionId sessionId, CancellationT
                 .GroupBy(l => l.Id);
             return locations;
         }
+
+        private static IEnumerable<IGrouping<SourceId, SourceLocation>> GetBPReqNextLocation(DebugStore store, BreakpointRequest req)
+        {
+            var nextLocation = store.FindBreakpointNextLocation(req);
+            var nextLocationList = new List<SourceLocation>();
```

Comment:
can we use a `stackalloc SourceLocation[1]` here?

Reply:
It looks like we cannot do it for managed types:
error CS0208: Cannot take the address of, get the size of, or declare a pointer to a managed type ('SourceLocation')

I changed to:
var nextLocationList = new SourceLocation[1];